cfme/services/myservice/ssui.py

```diff
@@ -230,14 +230,15 @@ def launch_vm_console(self, catalog_item):
 @MiqImplementationContext.external_for(MyService.retire, ViaSSUI)
 def retire(self):
     view = navigate_to(self, 'Retire')
-    view.retire.click()
-    view = self.create_view(MyServicesView)
-    assert wait_for(
-        lambda: view.is_displayed, delay=3, num_sec=300,
-        message="waiting for view to be displayed"
-    )
-    # TODO - remove sleep when BZ 1518954 is fixed
-    time.sleep(10)
+    if self.appliance.version <= "5.9":
+        view.retire.click()
+        view = self.create_view(MyServicesView)
```

Comment:
Required:
I think you won't need it; as your navigation `(L232)` returning you same view.